<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reversigui</class>
 <widget class="QMainWindow" name="reversigui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>reversigui</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/blacktile.png</normaloff>:/icons/blacktile.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="gameNameLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Serif</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>REVERSI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nová hra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadGameButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Načíst hru</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nápověda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Konec</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="helpMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>701</width>
      <height>351</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Nápověda pro hru reversi&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Pravidla:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Hraje se na desce o velikosti 6x6, 8x8, 10x10 nebo 12x12 polí.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Pozice na desce jsou označeny čísly 1 až 6/8/10/12 a znaky A až F/H/J/L.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Každý hráč ve svém tahu položí kámen své barvy na prázdné pole tak, aby mezi tímto kamenem a jiným kamenem stejné barvy byl alespoň jeden kámen druhého hráče. Hráč s nejvyšším počtem kamenů vyhrává.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Použití aplikace:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Je možné hrát buď proti počítači nebo druhému hráči. Pokládání kamenů probíhá kliknutím na požadovanou pozici. Lze využít možností undo a redo pro pohyb v jednotlivých krocích hry.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Černé kameny jsou prvního hráče, bílé kameny druhého. &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;V herním poli jsou červenou barvou označeny pozice, na které je možno položit v daném tahu kámen.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToMenuButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zpět</string>
    </property>
   </widget>
   <widget class="QPushButton" name="onePlayerButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jeden hráč</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twoPlayersButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dva hráči</string>
    </property>
   </widget>
   <widget class="QLabel" name="diffChooseLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vyberte obtížnost</string>
    </property>
   </widget>
   <widget class="QPushButton" name="easyAiButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lehký</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hardAiButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Těžký</string>
    </property>
   </widget>
   <widget class="QLabel" name="boardsizeLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vyberte velikost herní plochy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x6SizeButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6x6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x8SizeButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8x8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x10SizeButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10x10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x12SizeButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12x12</string>
    </property>
   </widget>
   <widget class="QLabel" name="alphaLocLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>951</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="numLocLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>41</width>
      <height>631</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>191</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Loma</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="undoButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redoButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
   <widget class="QLabel" name="turnLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="quitGameButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ukončit hru</string>
    </property>
   </widget>
   <zorder>helpMessage</zorder>
   <zorder>gameNameLabel</zorder>
   <zorder>loadGameButton</zorder>
   <zorder>helpButton</zorder>
   <zorder>exitButton</zorder>
   <zorder>backToMenuButton</zorder>
   <zorder>twoPlayersButton</zorder>
   <zorder>diffChooseLabel</zorder>
   <zorder>newGameButton</zorder>
   <zorder>hardAiButton</zorder>
   <zorder>easyAiButton</zorder>
   <zorder>onePlayerButton</zorder>
   <zorder>boardsizeLabel</zorder>
   <zorder>x6SizeButton</zorder>
   <zorder>x8SizeButton</zorder>
   <zorder>x10SizeButton</zorder>
   <zorder>x12SizeButton</zorder>
   <zorder>alphaLocLabel</zorder>
   <zorder>numLocLabel</zorder>
   <zorder>scoreLabel</zorder>
   <zorder>undoButton</zorder>
   <zorder>redoButton</zorder>
   <zorder>turnLabel</zorder>
   <zorder>quitGameButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>reversigui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitGameButton</sender>
   <signal>clicked()</signal>
   <receiver>reversigui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
